Core.lua

```diff
@@ -673,6 +673,17 @@ function R:IsAttemptAllowed(item)
 		return false
 	end
 
+	local _, _, _, _, _, _, _, _, _, LfgDungeonID = GetInstanceInfo()
+	print("Torghast Layer " .. LfgDungeonID)
```

Comment:
If this is meant to remain as debug output, it should use `Rarity.Debug` so that it can be hidden.

Otherwise, it'll have to be removed so as to avoid spamming the chat frame :)

Reply:
ill probably remove that one since its abit spammy but yeah i also forgot to change it from print to debug, thanks